libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.h

```diff
@@ -289,19 +289,40 @@ NS_SWIFT_NAME(SalesforceManager)
 - (nonnull NSString *)devInfoTitleString;
 
 /**
- * Returns the ScreenLockManager instance.
+ * Returns The ScreenLockManager instance.
  *
- * @return the Screen Lock Manager
+ * @return The Screen Lock Manager.
  */
 - (id <SFScreenLockManager>)screenLockManager;
 
 /**
- * Returns the BiometricAuthenticationManager instance.
+ * Returns The BiometricAuthenticationManager instance.
  *
- * @return the Biometric Authentication Manager
+ * @return The Biometric Authentication Manager.
  */
 - (id <SFBiometricAuthenticationManager>)biometricAuthenticationManager;
 
+/**
+ * Creates the NativeLoginManager instance.
+ *
+ * @param remoteAccessConsumerKey The Conneted App consumer key.
```

Comment:
Totally random, but I noticed `Connected` is misspelled while I was just reading for context in the upcoming OTP effort.  There are a couple of instances.